libkbfs/conflict_renamer.go

```diff
@@ -60,18 +67,11 @@ func splitExtension(path string) (string, string) {
 	return path, ""
 }
 
-func newWriterInfo(ctx context.Context, cfg Config, uid keybase1.UID,
-	key kbfscrypto.VerifyingKey,
-	revision MetadataRevision) (writerInfo, error) {
-	ui, err := cfg.KeybaseService().LoadUserPlusKeys(ctx, uid)
-	if err != nil {
-		return writerInfo{}, err
-	}
-
+func newWriterInfo(uid keybase1.UID, key kbfscrypto.VerifyingKey,
+	revision MetadataRevision) writerInfo {
```

Comment:
For posterity, some justification for this, as I have vague memories of pushing for the old version (do all the device name loading at `newWriterInfo` time). That was okay when conflict resolution was the only thing using `crChains`, but it seems like `crChains` is a more general-purpose data structure, and those other use cases don't need to do any resolving, so it makes more sense to do the device name loading on-demand.